guide/src/watch-command.md

```diff
@@ -0,0 +1,21 @@
+# Watch command
+
+Yarner has a subcommand `watch` to automatically build a project after files changed. This may be particularly convenient when switching forth and back between Markdown editing and code editing with reverse mode.
+
+<div style="border: 2px solid red; padding: 0.5em;">
+Warning: This feature is still experimental and modifies the original Markdown sources. Make a backup of the sources before using it!
```

Comment:
If I understand things correctly, it will also currently not watch `Yarner.toml` and hence not react to changes to the set of source files?

Reply:
It does watch `Yarner.toml`, but only as source file. I.e. it will attempt a forward pass on changes.

A limitation is still that changes in the list of files to watch are not taken into account (e.g. if a transclusion is added).

Reply:
When returning to use a closure, it might be possible to modify the watchers? Will give it a try...

Reply:
> A limitation is still that changes in the list of files to watch are not taken into account (e.g. if a transclusion is added).

I don't think implementing this in this PR here would be helpful. Just, that it could be included in the documentation.